src/main/cli/ocrService.ts

```
@@ -0,0 +1,395 @@
+          if (error instanceof OcrRuntimeBusyError) {
+            throw new CliRequestError('conflict', error.message, { httpStatus: 409 })
+          }
+          this.log.warn('[CLI] OCR cache clear failed', {
+            failure: { name: error instanceof Error ? error.name : typeof error }
+          })
+          throw new CliRequestError('unavailable', 'OCR cache could not be cleared', {
+            httpStatus: 503,
+            retriable: true
+          })
+        }
+        const status = await this.options.ocrRuntime.getStatus()
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)